config/database.yml

```diff
@@ -9,7 +9,7 @@ default: &default
   encoding: utf8
   pool: 5
   username: root
-  password: 
+  password: pass
```

Comment:
čia visų gali skirtis, pvz mano be passwordo, reikėtų pasidomėti kaip pasidaryti tik lokalius šiuos nustatymus, o į production tik default kokie nors keliautų. Aš ir pats dar nežinau kaip ant rails tai padaryti, reikės patikrinti